pkg/deprecatedflags/deprecatedflags.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package deprecatedflags can be used to declare a flag that is no
+// longer supported.
+package deprecatedflags
+
+import (
+	"flag"
+
+	"k8s.io/klog"
+)
+
+// Add defines a deprecated option which used to take some kind of
+// value. The type of the argument no longer matters, it gets ignored
+// and instead using the option triggers a deprecation warning. The
+// return code can be safely ignored and is only provided to support
+// replacing functions like flag.Int in a global variable section.
+func Add(name string) bool {
```

Comment:
why do these need a return value? can they just return nothing instead?

I'm thinking maybe this was a creative solutiont there is a variable you can set to `_` when you use these functions. If that is the case and it is truly necessary, lets at least document that in a comment here so that ppl in the future aren't confused

Reply:
Yes, that's what it is for. `var _ = something()` doesn't work unless `something()` returns a value.

I tried to explain that in the last sentence of the comment ("is only provided to support
replacing functions like flag.Int in a global variable section"). Would an example be clearer?

Reply:
oh i just have questionable reading comprehension skills. that makes sense. Thanks